src/main/kotlin/no/nav/familie/ef/sak/api/oppgave/OppgaveController.kt

```diff
@@ -13,13 +15,19 @@ import org.springframework.web.bind.annotation.*
 @RequestMapping("/api/oppgave")
 @ProtectedWithClaims(issuer = "azuread")
 @Validated
-class OppgaveController(private val oppgaveService: OppgaveService, private val tilgangService: TilgangService) {
+class OppgaveController(private val oppgaveService: OppgaveService,
+                        private val tilgangService: TilgangService,
+                        private val pdlClient: PdlClient) {
+
+    private val secureLogger = LoggerFactory.getLogger("secureLogger")
 
     @PostMapping(path = ["/soek"],
                  consumes = [MediaType.APPLICATION_JSON_VALUE],
                  produces = [MediaType.APPLICATION_JSON_VALUE])
     fun hentOppgaver(@RequestBody finnOppgaveRequest: FinnOppgaveRequestDto): Ressurs<FinnOppgaveResponseDto> {
-        return Ressurs.success(oppgaveService.hentOppgaver(finnOppgaveRequest.tilFinnOppgaveRequest()))
+        val aktørId = finnOppgaveRequest.ident?.let { pdlClient.hentAktørId(it).hentIdenter.identer.first().ident }
```

Comment:
Tar gjerne en test der hvor finnOppgaveRequest.ident er tom streng (`""`) 

Litt usikker på om koden tar høyde for det nå?

Reply:
Nja. Mulig det ville fungere, men det vil jo være avh. av oppgave/pdl. Jeg skriver om. Vi trenger jo ikke plage pdl med queries for tomme strenger.

Reply:
Lagt på takeUnless i hentOppgaver
val aktørId = finnOppgaveRequest.ident.takeUnless { it.isNullOrBlank() }
                ?.let { pdlClient.hentAktørId(it).hentIdenter.identer.first().ident }
og to tester for null og " " verdier fra ui